transform-chunk-sizes.test.ts

```diff
@@ -9,7 +9,7 @@ import { TransformChunkSizes } from "./transform-chunk-sizes.ts";
  */
 class NumberSource extends ReadableStream<Uint8Array> {
   constructor(delayMs: number, chunksCount: number, bytesPerChunk = 1) {
-    let intervalTimer: number;
+    let intervalTimer: any;
```

Comment:
```suggestion
    let intervalTimer: ReturnType<typeof setTimeout>;
```

Better to use a specific type.

Reply:
Ok fixed